src/test/java/edu/harvard/iq/dataverse/api/AdminIT.java

```diff
@@ -671,4 +679,30 @@ public void testMigrateHDLToDOI() {
                 .statusCode(OK.getStatusCode());
     }
 
+    @Test
+    public void testLoadMetadataBlock_ErrorHandling() {
+        Response createUser = UtilIT.createRandomUser();
+        String apiToken = UtilIT.getApiTokenFromResponse(createUser);
+
+        byte[] updatedContent = null;
+        try {
+            updatedContent = Files.readAllBytes(Paths.get("src/test/resources/tsv/test.tsv"));
+        } catch (IOException e) {
+            logger.warning(e.getMessage());
+            assertEquals(0,1);
+        }
+        Response response = UtilIT.loadMetadataBlock(apiToken, updatedContent);
+        assertEquals(500, response.getStatusCode());
+        response.then().assertThat().statusCode(INTERNAL_SERVER_ERROR.getStatusCode());
+
+        String body = response.getBody().asString();
+        String status = JsonPath.from(body).getString("status");
+        assertEquals("ERROR", status);
+
+        String message = JsonPath.from(body).getString("message");
+        assertEquals(
+          "Error parsing metadata block in DATASETFIELD part, line #5: missing 'watermark' cell (#5)",
```

Comment:
This test is failing because the test has "cell" in the error message while the api uses "column. Suggest putting the error message into the Bundle. None of the rest of the endpoints use the bundle because this is a very old api, but we're trying to add new work to the bundle when possible

Reply:
Oh, and you'll need to get the latest from the develop branch.